addons/sale_project/models/project.py

```diff
@@ -278,7 +278,7 @@ def get_action(sol_id):
         return [{
             **sol_read,
             **get_action(sol_read['id']),
-        } for sol_read in sols.with_context(with_price_unit=True).read(['display_name', 'product_uom_qty', 'qty_delivered', 'qty_invoiced', 'product_uom'])]
```

Comment:
I am unsure that we should remove this context key... I have asked @aju-odoo 
Let's wait for her input.